doc/releases/pacific.rst

```diff
@@ -2,6 +2,239 @@
 Pacific
 =======
 
+v16.2.15 Pacific
+================
+
+This is the fifteenth backport release in the Pacific series.
+
+Notable Changes
+---------------
+
+* `ceph config dump --format <json|xml>` output will display the localized
+  option names instead of its normalized version. For e.g.,
+  "mgr/prometheus/x/server_port" will be displayed instead of
+  "mgr/prometheus/server_port". This matches the output of the non pretty-print
+  formatted version of the command.
+
+* CEPHFS: MDS evicts clients which are not advancing their request tids which causes
```

Comment:
`CEPHFS: MDS evicts clients who are not advancing their request tids, which ...`

(swap first `which` with `who`)

Reply:
... and CEPHFS with CephFS, if we are nit picking ;)

Reply:
Agreed.